fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/Identifier.java

```diff
@@ -59,7 +60,11 @@ public boolean isQualified() {
 
     @Override
     public String toString() {
-        return String.join(".", qualifier) + (qualifier.isEmpty() ? "" : ".") + name;
+        return qualifier.stream()
+                .map(q -> "\"" + q + "\"")
+                .collect(Collectors.joining("."))
+                + (qualifier.isEmpty() ? "" : ".")
+                + "\"" + name + "\"";
```

Comment:
Using double quotes in error messages can lead to problems when integrating with logging frameworks. For example, if the user prints out the error message to a logging system with key-value pairs of quoted keys and values, then the internal quote within the message can end up confusing things. We could get around this by using single quotes (or backticks), though that is different from what the SQL query coming in will do